recipe/meta.yaml

```diff
@@ -7,10 +7,12 @@ package:
 source:
   fn: sympy-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/s/sympy/sympy-{{ version }}.tar.gz
-  md5: c410a9c2346878716d16ec873d72e72a
+  sha256: ac5b57691bc43919dcc21167660a57cc51797c28a4301a6144eff07b751216a4
+  patches:
+  - 13276.patch
```

Comment:
Just a suggestion but maybe you can rename this to `backport_pr13276.patch`, or maybe add a comment line above it?